tests/engine/artifact_filters.py

```diff
@@ -81,7 +81,7 @@ def testBuildFindSpecsWithFileSystem(self):
     test_filter_file.BuildFindSpecs(
         environment_variables=[environment_variable])
     find_specs_per_source_type = knowledge_base.GetValue(
-        test_filter_file._KNOWLEDGE_BASE_VALUE)
+        test_filter_file.KNOWLEDGE_BASE_VALUE)
```

Comment:
just override pylint check for access to protected members in the tests

Reply:
I do access this in winreg.py as well